templates/ruby/lib/container/payload.rb.twig

```diff
@@ -0,0 +1,77 @@
+module Appwrite
+  class Payload
+    attr_reader :data
+    attr_reader :filename
+
+    def initialize(data: nil, path: nil, filename: nil)
```

Comment:
Do we need path here?

Reply:
It is stored to enable loading the file in chunks